modules/configuring-external-databases/proc-configuring-pvc-mounts.adoc

```diff
@@ -0,0 +1,82 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-configuring-pvc-mounts_{context}"]
+= Configuring Persistent Volume Claim (PVC) mounts
+
+You can configure which containers the Persistent Volume Claim (PVC) mounts to by adding `rhdh.redhat.com/containers` to your configuration file. You can configure multiple secrets by using annotations to specify mount paths for each secret and target specific containers where the secrets should mount. Adding annotations ensures flexible storage and secrets management across different containers. The `default-config/deployment.yaml` file defines the dynamic plugins directory, as shown in the following example:
```

Comment:
...by adding `rhdh.redhat.com/containers` **annotation**...

Reply:
"You can configure multiple secrets by using annotations to specify mount paths for each secret and target specific containers where the secrets should mount."

It is correct but related to Secret not PVC